### ISSUE TITLE
[COGNITION]: Footer links are inconsistently styled and missing interactive cues

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3179]
[COGNITION]: Footer links are inconsistently styled and missing interactive cues

## Description

Default links have cues for all states of interactivity: `a:link, a:focus, a:hover, a:visited, a:active`, but footer links are inconsistently styled (See [design.va.gov>Typography>LInks](https://design.va.gov/design/typography#links))

In the enclosed screenshot of a narrower viewport, the bottom list of links all have `text-decoration: underline` applied, but the other links have no visual indication they are interactive. In addition, at narrower viewports, none of the footer links have hover styling. For wider viewports, the underlined links have interactive hover styling. Focus styling is provided.

## Point of Contact

Jennifer

## Acceptance Criteria

- As a user, I want to view links in the footer of VA.gov and visually understand what items are links.
- As a mouse user, I want to hover over links in the footer of VA.gov and visually understand which items are interactive, regardless of browser viewport width.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Navigate to VA.gov
2. Reduce the width of the browser window (or visit on mobile)
3. Open the Contact Us accordion, distinguish what content is interactive; Compare with the remaining links in the footer
4. Using the mouse, distinguish what text is interactive
5. Increase the width of the browser window to discern what content is interactive

### Current CSS
<img width="343" alt="footer-links-styling-hover" src="https://user-images.githubusercontent.com/57469/68327751-f0511d00-009b-11ea-93d6-5ed0dcc8024e.png">

### Suggested CSS
```
.va-footer-accordion-content>.va-footer-links>li>a {
        text-decoration: underline;
}
.va-footer-accordion-content>.va-footer-links>li>a:hover {
        color: #fdb81e;
}
```

## WCAG or Vendor Guidance (optional)

* [WCAG Success Criterion 1.4.1 Use of Color](https://www.w3.org/TR/WCAG21/#use-of-color)
* [WEBAIM: Link Appearance](https://webaim.org/techniques/hypertext/link_text#appearance)

**Comment by alexpappasoddball** [comment]
@brandonrapp This one is coming your way

**Comment by meganhkelley** [comment]
@brianalloyd I'm not sure if this is still an issue, but if so I imagine it would fall to your team to investigate

**Comment by brianalloyd** [comment]
Thanks @meganhkelley, we'll validate if it's still a current issue and take for action. Thanks.

**Comment by brianalloyd** [comment]
@RLHecht and @lunascoop could you take a look at this ticket when time permits and validate if this is still an issue that we need to consider for future action.

**Comment by RLHecht** [comment]
@brianalloyd I think this is either an FE or a design ticket. I'm not sure I can tell if the links are still not styled consistently.

**Comment by brianalloyd** [comment]
Validated, closing ticket.
